confignode/src/main/java/org/apache/iotdb/confignode/persistence/schema/ClusterSchemaInfo.java

```diff
@@ -202,7 +202,7 @@ public StorageGroupSchemaResp getMatchedStorageGroupSchemas(GetStorageGroupPlan
       LOGGER.error("Error StorageGroup name", e);
       result.setStatus(
           new TSStatus(TSStatusCode.STORAGE_GROUP_NOT_EXIST.getStatusCode())
-              .setMessage("Error StorageGroup name"));
+              .setMessage("Error StorageGroup name: " + e));
```

Comment:
`+ e.getMessage()`? I think `getMessage()` is more friendly for human read.

Reply:
Fixed~